tests/foreman/api/test_operatingsystem.py

```diff
@@ -63,21 +63,16 @@ def test_verify_bugzilla_1114640(self):
         os_param = entities.OperatingSystemParameter(
             name=name, operatingsystem=1, value=value
         ).create()
-        self.assertEqual(os_param.name, name)
-        self.assertEqual(os_param.value, value)
+        assert os_param.name == name
+        assert os_param.value == value
 
 
-class OperatingSystemTestCase(APITestCase):
+class TestOperatingSystem:
     """Tests for operating systems."""
 
-    @classmethod
-    def setUpClass(cls):
-        """Create an organization which can be re-used in tests."""
-        super().setUpClass()
-        cls.org = entities.Organization().create()
-
     @pytest.mark.tier1
-    def test_positive_create_with_name(self):
+    @pytest.mark.parametrize('name', **parametrized(valid_data_list()))
```

Comment:
Please add `:parametrized: yes` to test function docblocks when they are parametrized.

Reply:
Done.